src/Admin/Controllers/ToolsController.cs

```diff
@@ -465,8 +469,9 @@ public async Task<IActionResult> StripeSubscriptions([FromForm] StripeSubscripti
     {
         if (!ModelState.IsValid)
         {
+            var isProduction = _environment.IsProduction();
             model.Prices = (await _stripeAdapter.PriceListAsync(new Stripe.PriceListOptions() { Limit = 100 })).Data;
-            model.TestClocks = await _stripeAdapter.TestClockListAsync();
+            model.TestClocks = isProduction ? new List<Stripe.TestHelpers.TestClock>() : await _stripeAdapter.TestClockListAsync();
```

Comment:
Suggestion: passing `isProduction` as a `TestClockListAsync` method parameter, bullet proofing it in case it gets used elsewhere.